templates/download/raspberry-pi/index.html

```diff
@@ -60,48 +60,64 @@ <h2>First time installing Ubuntu on Raspberry Pi?</h2>
     </div>
   </div>
 </section>
-
 <section class="p-strip is-shallow u-hide--small">
   <div class="row">
-    <div class="col-6 col-medium-3 u-vertically-center">
-      <h2>Ready for Raspberry Pi 5</h2>
+    <div class="col-6 col-medium-3">
+      <span class="p-text--x-small-capitalised u-no-margin--bottom">Ready for Raspberry Pi 5</span>
+      <h2>Ubuntu {{ releases.latest.full_version }}</h2>
     </div>
-    <div class="col-6 col-medium-3 u-vertically-center u-align--right">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/953685e6-raspberry-pi-5.png",
-        alt="",
-        width="300",
-        height="200",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
+    <div class="col-6 col-medium-3 u-vertically-center u-align--right u-align-text--left">
+      <p>The Raspberry Pi 5 is supported on the latest development release of Ubuntu with nine months of support, until
+        July 2024.</p>
+      <p>
+        <a href="/download/raspberry-pi/thank-you?version={{ releases.latest.full_version }}&amp;architecture=desktop-arm64+raspi"
+          onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 64-bit - Ubuntu Desktop', 'eventLabel' : '{{ releases.latest.full_version }}', 'eventValue' : undefined });"
+          class="p-button u-no-margin--bottom">Download Ubuntu Desktop 23.10</a>
+      <p><em>* Minimum 4GBs of RAM required</em></p>
+      </p>
+      <p>
+        <a href="/download/raspberry-pi/thank-you?version={{ releases.latest.full_version }}&amp;architecture=server-arm64+raspi"
+          onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 64-bit - Ubuntu Server', 'eventLabel' : '{{ releases.latest.full_version }}', 'eventValue' : undefined });"
+          class="p-button">Download Ubuntu Server 23.10</a>
+      </p>
     </div>
   </div>
 </section>
 
 <section class="p-strip u-no-padding--top">
```

Comment:
you don't need a separate section here, it can just another row within the previous section